roles/kubernetes-apps/metallb/templates/metallb.yml.j2

```
@@ -1755,14 +1765,15 @@ spec:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
-{% if metallb_config.controller.tolerations %}
+{% if metallb_config.controller.tolerations is defined %}
```

This will fail with undefined error if `metallb_config` doesn't contain `controller` (as the example docs show for example). It can be worked around by just adding `controller: {}` but I suggest you handle it here.

```suggestion
{% if metallb_config.controller is defined and metallb_config.controller.tolerations is defined %}
```